### PR TITLE
Add frame download option

### PR DESCRIPTION
**Pull request by hputzek**
Hey, please try if it works.

I tested it and had the same output in a hex editor as in the hex view of your testing tool so I hope its ok like that.

### CODE REVIEW

**Comment by hputzek** [comment]
Oh @martinberlin btw sorry I did "autoformat" in my IDE thats why there are so many changes.
The important part is the new function for downloading, I'll mark that in the code for you

File: examples/test/binary.js [hunk 1]
```diff
@@ -17,239 +18,296 @@ var stripe = $("div#stripe"),
...
-  tl1.duration(delay.val());
-  tl1.play(0);
+
+/**
+ * Creates a file out of the given body parameter and triggers a download of the created file
+ * @param body the payload to add to the file
+ * @param filename
+ * @param extension (of the file)
+ */
+function createAndDownloadBlobFile(body, filename, extension = 'bin') {
```
Comment (hputzek) [inline examples/test/binary.js:281]: @martinberlin  this is the download func

**Comment by martinberlin** [comment]
Awesome. I will test and merge this morning. Thanks a lot!
Proposed table:
Size in bytes | uncompress millis | estimated Max FPS (adding 9ms Neopixel work to send data)

**Comment by hputzek** [comment]
The form state is now persisted to localstorage, this means if you for example change the IP you do not need to re-enter it on every pageload :-)

**Review by martinberlin — APPROVED** [review]

**Comment by martinberlin** [comment]
Compared to the version on master that was the old design with the RGBW option this is not working as before and adding some strange colors. 
Something went bad with the addition of the binary download. I will try to check it line by line and report maybe at the same time I find the reason of the 1 byte missing. Nice design though ;)

Solved the colors parts. There is just a JS error coming:
TypeError: formData is null

After messing some time with it I still cannot fix it. So I leave it in develop as it was before until I find time to deal with it and if not, I will just leave design as is and try to implement the old JS with your additions. But please next time you edit part of html or js do not Lint it until we agreeded before to. 
Like this is very hard to detect what changed. I prefer more ugly code than nicer formatted but non working.

**Pull request merged**